### PR TITLE
Convert nav into react component

### PR DESCRIPTION
**Pull request by NMilivojevic**
Should test the conditionals regarding if the user is logged in, is admin, is partner, etc, to see if I have successfully converted that part. Added #header-root in header part of master pug file

### CODE REVIEW

File: js/react/components/nav/index.jsx [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+										Admin Panel
+									</a>
+								</li>
+							) : (
+								<li className="nav-item">
+									<a className="nav-link" href="/profile">
+										Profile
+									</a>
+								</li>
+							)}
```
Comment (Enubia) [inline js/react/components/nav/index.jsx:68]: is it possible to do this condition without nested ternary expressions?
it's really hard to read and understand in this way

File: js/react/components/nav/index.jsx [hunk 2]
```diff
@@ -0,0 +1,154 @@
+import PropTypes from 'prop-types';
+
+function Nav(props) {
```
Comment (Enubia) [inline js/react/components/nav/index.jsx:3]: please try to create a new navbar with this in mind 
https://getbootstrap.com/docs/5.3/components/navbar/#responsive-behaviors

right now we have two separate ones but bootstrap is able to use only one navbar and collapse it on mobile

**Review by Enubia — CHANGES_REQUESTED** [review]

File: example-config.env [hunk 3]
```diff
@@ -1,44 +0,0 @@
-PORT=3000
-
```
Comment (Enubia) [inline example-config.env:2]: why was this file deleted?

File: js/react/components/nav/index.jsx (hunk 1 above)
Comment (NMilivojevic) [reply js/react/components/nav/index.jsx]: Maybe with explicit && conditionals. 
Screenshot from 2023-08-09 14-39-22

File: example-config.env (hunk 3 above)
Comment (NMilivojevic) [reply example-config.env:2]: Forgot to copy it and then rename it, my bad

**Review by Enubia — APPROVED** [review]
LGTM!
